### PR TITLE
feat: adding normalization

### LINKED ISSUE #12: Inconsistency in outputting the git url
I see that the code tries to get the url from your .git/config file. However if I use SSH to clone the repository url and another person uses https to clone the repository, then there would be inconsistencies in setting the "GitUrl" tag on the aws resources.

In the case of SSH the end result would be:

```
     {
      "Key": "GitUrl",
      "Value": "git@github.com:user/cdk-repo.git"
     },
```

And in case of https:

```
     {
      "Key": "GitUrl",
      "Value": "https://github.com/user/cdk-repo.git"
     },
```

Would it be an idea to turn it into the https url by default?

### PR DESCRIPTION
**Pull request by mbonig**
Fixes #12 and #9

### CODE REVIEW

File: src/GitUrlTagger.ts [hunk 1]
```diff
@@ -5,20 +5,32 @@ import { IConstruct } from 'constructs';
...
+   */
   readonly tagName?: string;
+
+  /**
+   * A flag on whether to try to normalize the URL found in the git config
+   * If enabled, it will turn ssh urls into https urls.
+   *
+   * @default false
+   */
+  readonly normalizeUrl?: boolean;
```
Comment (mikejgray) [inline src/GitUrlTagger.ts:20]: Consider defaulting to true, as the UX is probably better with HTTPS urls vs. SSH urls

File: src/GitUrlTagger.ts [hunk 2]
```diff
@@ -51,4 +63,13 @@ export class GitUrlTagger implements IAspect {
     }
     return 'No Code Repo Found';
   }
+
+  private normalizeUrl(gitUrl: string) {
+    const regexSSH = /^git@([A-Za-z0-9-]+\.[A-Za-z0-9-]+):([A-Za-z0-9-]+)\/([A-Za-z0-9-]+)\.git$/;
```
Comment (mikejgray) [inline src/GitUrlTagger.ts:68]: Now we have two problems!

**Review by mikejgray — APPROVED** [review]
Nice! A good mock for the soul

File: src/GitUrlTagger.ts (hunk 1 above)
Comment (mbonig) [reply src/GitUrlTagger.ts:20]: I agree, changing it now...

File: src/GitUrlTagger.ts (hunk 2 above)
Comment (mbonig) [reply src/GitUrlTagger.ts:69]: I don't know what you mean...

**Review by mikejgray — APPROVED** [review]
:shipit:

**Pull request merged**